src/poker.js

```diff
@@ -1,10 +1,85 @@
-function winningPair() {}
+const cardMap = {
+  3: 3,
+  4: 4,
+  5: 5,
+  6: 6,
+  7: 7,
+  8: 8,
+  9: 9,
+  10: 10,
+  J: 11,
+  Q: 12,
+  K: 13,
+  A: 14
+}
+
+function winningPair(pairOne, pairTwo) {
+  if (pairOne[0] !== pairOne[1] && pairTwo[0] !== pairTwo[1]) return []
+
+  if (pairOne[0] === pairOne[1] && pairTwo[0] === pairTwo[1]) {
```

Comment:
You could make this easier to read and reduce the amount of code by saving a boolean comparison into a variable:

```js
const hand1IsPair = pairOne[0] === pairOne[1]
const hand2IsPair = pairTwo[0] === pairTwo[1]

if (!hand1IsPair && !hand2IsPair) {}
```